static/sass/_pattern_ratings.scss

```diff
@@ -0,0 +1,95 @@
+.p-rating {
+  background-image: url("#{$assets-path}19c4273e-level+of+difficulty.svg");
+  background-position-x: -75px;
+  background-position-y: center;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: $sp-large;
+  margin-left: $sp-xx-small;
+  position: relative;
+  text-indent: -9999px;
+  width: 74px;
+}
```

Comment:
Here I don't think you need the `::before`.
If we look at the svg we can see it's made so it can represent any "out of 5" rating if we just offset the background
![image](https://user-images.githubusercontent.com/11927929/99688750-71503c00-2a86-11eb-8d4a-9a62c67951d2.png)

So I would get rid of the `::before` pseudo element and implement it like this
```css
.p-rating--1 {
  background-image: url("#{$assets-path}19c4273e-level+of+difficulty.svg");
  background-position-x: calc(-75px + (15px * 1));
  background-position-y: center;
  background-repeat: no-repeat;
  display: inline-block;
  height: $sp-large;
  width: 71px;
}

.p-rating--2 {
  background-image: url("#{$assets-path}19c4273e-level+of+difficulty.svg");
  background-position-x: calc(-75px + (15px * 2));
  ...
}
.p-rating--3 {
  background-image: url("#{$assets-path}19c4273e-level+of+difficulty.svg");
  background-position-x: calc(-75px + (15px * 3));
  ...
}

etc.
```
If we set the width to `71px` we can also get rid of the left margin. (71px because there are five 11px wide squares and four 4px spaces)

What do you think?

Reply:
Scratch that because of half ratings